pkg/controller/perconaxtradbcluster/perconaxtradbcluster_controller.go

```diff
@@ -264,6 +283,22 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return fmt.Errorf("create PXC Service: %v", err)
 		}
+
+		// PodDistributedBudget object for ProxySQL
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: proxySet.Name, Namespace: proxySet.Namespace}, proxySet)
+		if err == nil {
+			pdbProxySQL := pxc.NewPodDistributedBudget(cr, sfsProxy)
+
+			err = setControllerReference(proxySet, pdbProxySQL, r.scheme)
+			if err != nil {
+				return err
+			}
+
+			err = r.client.Create(context.TODO(), pdbProxySQL)
+			if err != nil && !errors.IsAlreadyExists(err) {
+				return fmt.Errorf("create PodDisruptionBudget-PXC: %v", err)
+			}
+		}
```

Comment:
Should be an appropriate error handling. What if `err != nil`?

Reply:
In this keys only two ways are posible:
1) if errors.IsAlreadyExists(err) -> Do nothing
2) if !errors.IsAlreadyExists(err) -> return err

I guess this part works correct. Isn't it?

Reply:
meant not lines 298, but error handling in the block L289-301:
starting here https://github.com/Percona-Lab/percona-xtradb-cluster-operator/pull/85/files/cc25b8ecd6d9bb4a91e7bd26778b7daf6452d1ab#diff-f446014a9240c7ae115b90dc2fa50e6fR289 
if 
```go
err = r.client.Get(context.TODO(), types.NamespacedName{Name: proxySet.Name, Namespace: proxySet.Namespace}, proxySet)
```
in L289 returns an error the Distributed budget creation will be skiped without any notices